samples/matter/common/src/bridge/util/bridge_util.h

```diff
@@ -125,21 +125,20 @@ template <typename T, std::size_t N> struct FiniteMap {
 	}
 	std::size_t FreeSlots() { return N - mElementsCount; }
 
-	bool ValueDuplicated(const T &value, uint16_t *key, uint8_t &numberOfDuplicates)
+	uint8_t GetDuplicates(const T &value, uint16_t *key)
```

Comment:
How about `GetDuplicatesCount or Number`? Currently the name suggest we would get e.g. list of duplicated objects.